### PR TITLE
Fix project filtering

### PR DESCRIPTION
**Pull request by jjidt**
Always pass projects in array to maintain sort order

### CODE REVIEW

File: web/src/client/js/components/Dashboard/DashboardComponent.js [hunk 1]
```diff
@@ -31,17 +31,15 @@ const DashboardComponent = ({
...
   const [showMyProjects, toggleMyProjects] = useState(false)
   const history = useHistory()
-  const objectKeys = Object.keys(projects)
-  const hasProjects = objectKeys.length > 0
-  const { myProjects, notMyProjects } = objectKeys.reduce((cur, key) => {
-    if (projects[key].createdBy === currentUserId) {
-      cur.myProjects[key] = projects[key]
+  const hasProjects = projects.length > 0
+  const { myProjects, notMyProjects } = projects.reduce((cur, project) => {
+    if (projects.createdBy === currentUserId) {
```
Comment (dheniges) [inline web/src/client/js/components/Dashboard/DashboardComponent.js]:
```suggestion
    if (project.createdBy === currentUserId) {
```

Drop the `s` on `projects` or this doesn't work

**Review by dheniges — CHANGES_REQUESTED** [review]
One small tweak for codekind

**Comment by jjidt** [comment]
updated

**Review by dheniges — APPROVED** [review]
🦖

**Pull request merged**